README.md

```diff
@@ -1,9 +1,37 @@
-# text-compare
+# text-compare 📝
+> A powerful React hook for comparing text with visual diffs and similarity scores
+> 
+**text-compare** is a lightweight yet powerful React hook that helps you compare two texts and visualize their differences. It provides customizable highlighting for added, removed, and common words, along with a similarity percentage score.
 
-A simple hook to compare two texts and provide you the diff and the matching percentages
 
-## To Use Components
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<a href="https://github.com/CreoWis/text-compare/fork" target="blank">
+<img src="https://img.shields.io/github/forks/CreoWis/text-compare?style=flat-square" alt="text-compare forks"/>
+</a>
+<a href="https://github.com/CreoWis/text-compare/stargazers" target="blank">
+<img src="https://img.shields.io/github/stars/CreoWis/text-compare?style=flat-square" alt="text-compare stars"/>
+</a>
+<a href="https://github.com/CreoWis/text-compare/issues" target="blank">
+<img src="https://img.shields.io/github/issues/CreoWis/text-compare?style=flat-square" alt="text-compare issues"/>
+</a>
+<a href="https://github.com/CreoWis/text-compare/pulls" target="blank">
+<img src="https://img.shields.io/github/issues-pr/CreoWis/text-compare?style=flat-square" alt="text-compare pull-requests"/>
+</a>
 
+## Features ✨
+
+- ⚡ Lightweight and performant
+- 🎨 Customizable highlighting colors
+- 📊 Similarity percentage calculation
+- 🔧 Easy to integrate
+
+## Basic Usage 🚀
+
+Here's a quick example to get you started:
+
+### To Use Components
```

Comment:
Remove this example for now.. will add when we add the component.

Reply:
Removed